.github/workflows/test_functional.yml

```diff
@@ -258,12 +258,17 @@ jobs:
             --exclude='*' \
             >rsyncout
           cat rsyncout
+
+      - name: Fiddle remote coverage paths
+        if: env.REMOTE_PLATFORM == 'true'
+        run: |
           # fiddle the python source location to match the local system
           for db in $(grep --color=never '.coverage\.' rsyncout); do
             sqlite3 "$db" "
               UPDATE file
               SET path = REPLACE(path, '/cylc/cylc/', '$PWD/cylc/')
-            "
+            " || \
+            (echo "::warning::${db}" && sqlite3 "$db" ".schema" && rm "$db")
```

Comment:
Just a thought, if sqlite3 commands are occasionally failing in the for loop, it might be the for loop that's the problem.

Perhaps if we `.join(':')` the SQL statements and issue them as one command we wouldn't have to ignore failures?